Modules/gcmodule.c

```diff
@@ -1072,10 +1072,22 @@ clear_freelists(PyInterpreterState *interp)
 {
     _PyTuple_ClearFreeList(interp);
     _PyFloat_ClearFreeList(interp);
-    _PyList_ClearFreeList(interp);
     _PyDict_ClearFreeList(interp);
     _PyAsyncGen_ClearFreeLists(interp);
     _PyContext_ClearFreeList(interp);
+#if defined(Py_GIL_DISABLED)
+    HEAD_LOCK(&_PyRuntime);
+    _PyThreadStateImpl *tstate = (_PyThreadStateImpl *)interp->threads.head;
+    while (tstate != NULL) {
+        _PyList_ClearFreeList(&tstate->freelist_state);
```

Comment:
When all free lists are migrated to `PyFreeListState, `I will wrap it as the utility function.
But not needed at this moment :)